sklearn/datasets/tests/test_openml.py

```diff
@@ -1018,27 +1017,7 @@ def test_fetch_openml_requires_pandas_error(monkeypatch, params):
         raise SkipTest("This test requires pandas to not be installed.")
 
 
-# TODO(1.4): move this parameter option in`test_fetch_openml_requires_pandas_error`
```

Comment:
For coverage, should we update this test and check for the `ImportError`?

Reply:
Uhm I thought that it was I did by moving this option in `test_fetch_openml_requires_pandas_error` but apparently this is not the case.

I will check what is the reason for not covering.